src/gt4py/next/otf/compilation/build_systems/compiledb.py

```diff
@@ -257,6 +257,8 @@ def _cc_create_compiledb(
     cmake_languages = [cmake_lists.Language(name="CXX")]
     if prototype_program_source.language is languages.Cuda:
         cmake_languages = [*cmake_languages, cmake_lists.Language(name="CUDA")]
+    elif prototype_program_source.language is languages.Hip:
+        cmake_languages = [*cmake_languages, cmake_lists.Language(name="HIP")]
```

Comment:
Optional suggestion. The current code is ugly and boring but I'm not sure if this suggestion actually improves anything:
```suggestion
    if (src_lang:=prototype_program_source.language) in [languages.Cuda, languages.Hip]: 
        cmake_languages = [*cmake_languages, cmake_lists.Language(name=src_lang.__name__.upper())]
```